Configuration/Settings.yaml

```diff
@@ -1,5 +1,11 @@
 Neos:
   Neos:
+    nodeTypes:
+      groups:
+        cookies:
+          position: 300
+          label: 'Cookies'
```

Comment:
Because in the whole extension no l10n translations are used, it isn't used here either.

Reply:
Yes, no worries. Translation is on the TODO-List. Will create an issue for that.